apps/pg-sync-service/src/lib/webhook.ts

```diff
@@ -99,10 +100,34 @@ export class AirtableWebhook {
     await this.rateLimiter.acquire();
     const response = await this.axiosInstance.get<ListWebhooksApiResponse>(
       `/bases/${this.baseId}/webhooks`,
-    ).catch((error) => {
-      const e = new Error(`Failed to list webhooks for base ${this.baseId}. Check your token has webhook:manage permissions.`, { cause: error });
-      logger.error(e);
-      throw e;
+    ).catch(async (error) => {
+      const webhookListError = `Failed to list webhooks for base ${this.baseId}`;
+      if (isAxiosError(error)) {
+        const errorDetails = {
+          baseId: this.baseId,
+          statusCode: error.response?.status,
+          errorType: error.response?.data?.error?.type,
+          errorMessage: error.response?.data?.error?.message,
+          feedbackMessage: '',
+        };
+
+        if (error.response?.status === 401) {
+          errorDetails.feedbackMessage = 'Check that your Airtable PAT is valid and has not expired';
+        } else if (error.response?.status === 403) {
+          errorDetails.feedbackMessage = 'Check that your Airtable PAT has the webhook:manage scope';
+        } else if (error.response?.status === 404) {
+          errorDetails.feedbackMessage = 'Check that the base exists and that you have access to it';
+        }
+
+        logger.error(`[WEBHOOK] ${webhookListError}: ${JSON.stringify(errorDetails)}`);
+        await slackAlert(env, [`[WEBHOOK] ${webhookListError}: ${JSON.stringify(errorDetails)}`]);
```

Comment:
I'm not sure about `JSON.stringify()` here (and elsewhere). It might make the information more difficult to read.